buildSrc/subprojects/kotlin-dsl/src/main/kotlin/org/gradle/kotlin/dsl/kotlin-dsl-upstream-candidates.kt

```diff
@@ -24,19 +24,23 @@ fun <T> deferred(value: () -> T): Any =
     Callable { value() }
 
 
-fun Project.execAndGetStdout(vararg args: String): String {
+fun Project.execAndGetStdout(workingDir: File, vararg args: String): String {
     val out = ByteArrayOutputStream()
     exec {
         isIgnoreExitValue = true
         commandLine(*args)
         standardOutput = out
+        this.workingDir = workingDir
     }
     val result = String(out.toByteArray()).trim()
     println("${args.toList()} result: $result")
```

Comment:
I guess this `println` should be removed in the final version.

Reply:
I saw several `git merge-base origin/master HEAD` returning `128`: https://builds.gradle.org/viewLog.html?buildId=16851319&buildTypeId=Gradle_Check_IndividualPerformanceScenarioWorkersLinux

I was trying to figure out what's going on. Removed.